stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -51,6 +51,13 @@
         <item name="streamUiAvatarOnlineIndicatorPosition">bottom</item>
     </style>
 
+    <style name="StreamUiChatInfoAvatarStyle" parent="StreamUiBaseAvatarStyle">
+        <item name="android:layout_width">@dimen/stream_ui_avatar_size_extra_large</item>
```

Comment:
Are you sure to keep `layout_width`, `layout_height`in style? It's not recommended to store layout dependent params in styles.